sphinxcontrib/spelling/tests/test_filter.py

```diff
@@ -34,3 +34,38 @@ def test_acronym_unicode():
     f = filters.AcronymFilter(t)
     words = [w[0] for w in f(text)]
     assert u'DBM' not in words, 'Failed to filter out acronym'
+
+
+def test_contributors():
+    f = filters.ContributorFilter(None)
+    names = [
```

Comment:
FWIW it's usually better to use `@pytest.mark.parametrize` to test individual cases vs stopping on the first failure.